src/Bpi/ApiBundle/Tests/WritingNodeTest.php

```diff
@@ -141,6 +154,29 @@ public function testPush()
             $this->assertTrue(in_array($nodeEntityTag->getTag(), $tags));
         }
         $this->assertCount(count($tags), $nodeEntityTags);
+
+        // Check asset upload.
+        $this->assertCount(1, $nodeEntity->getAssets()->getCollection());
+        // TODO: Take path from config.
+        $uploadsPath = $this->container->getParameter('kernel.project_dir').'/web/uploads/assets';
+        /** @var \Bpi\ApiBundle\Domain\Entity\File $asset */
+        foreach ($nodeEntity->getAssets()->getCollection() as $asset) {
+            $fileName = $uploadsPath.'/'.$asset->getName().'.'.$asset->getExtension();
+            $this->cleanupAssets[] = $fileName;
+            $this->assertFileExists($fileName);
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function tearDown()
+    {
+        parent::tearDown();
+
+        foreach ($this->cleanupAssets as $fileName) {
+            @unlink($fileName);
```

Comment:
Nice, that you do cleanup after the test, but please don't use the STFU operator (`@`)!
Use in `try … catch` to make it clear that/if you don't care about any errors.